daniel-alejandro-garcia/tarea-3/js/blackOrWhite.js

```diff
@@ -0,0 +1,20 @@
+function mineColor(file, rank) {
+    let result;
+    if(file =="a"|| file =="c"|| file == "e"|| file == "g") {
```

Comment:
@da-garcia , please read about Switch structure, this kind of validations should be handled with switch.